### ISSUE TITLE
TypeError: this.el is null when parsing dom nodes with no attributes

### ISSUE THREAD

**Issue by rydurham** [livewire/livewire#285]
**Describe the bug**
In certain conditions, the `this.el` will be null when referenced in the `extractTypeModifiersAndValue()` method (and elsewhere) in livewire.js. 

**To Reproduce**
1. Using Livewire v0.3.6, create a counter component as defined in the quick start docs.
2. Replace the contents of `counter.blade.php` with this: 

```html
<div style="text-align: center">
  <button wire:click="increment">+</button>
  @if ($count === 3)
    <p>This is a conditional dom node.</p>
  @endif
  <div> <!-- note the wrapping div -->
    <input type="text" name="count" id="count" wire:model="count">
  </div>
  <button wire:click="decrement">-</button>
</div>
```

3. View the counter component in a browser and increment the counter to 3.
4. You should see this error in the console:  

```
TypeError: this.el is null     [livewire.js:3277:7]
```

and the dom nodes below the wrapper div will not be rendered.

![Screenshot_2019-09-20 Livewire Error Demo](https://user-images.githubusercontent.com/674347/65368254-997aab80-dbf3-11e9-9d6b-e7a09f9e3e15.png)

**Expected behavior**
When the count increments to 3 the conditional dom node should display and the counter component should continue to be fully functional.

**Desktop:**
 - Browser: Firefox 69 
 - Ubuntu 18.04

**Additional context**
If you add an attribute to the wrapping div, such as `id="wrapper"`, everything seems to work as expected.

I have only recently started using Livewire; it is possible that I have missed some documentation that addresses this issue.  Thank you for bringing Livewire into the world - it is fantastic!

**Comment by calebporzio** [comment]
Thanks for the issue! I believe this is fixed in the latest release: v0.3.12

**Comment by rydurham** [comment]
Confirmed!  Thank you!
